.env

```diff
@@ -1,3 +1,5 @@
 REACT_APP_CHAIN_ID="43114"
 REACT_APP_NETWORK_URL="https://node.snowapi.net/ext/bc/C/rpc"
 REACT_APP_ANALYTICS_API="https://dev-axial-api.snowapi.net"
+REACT_APP_ANALYTICS_ENDPOINT='//dev-analytics.snowapi.net/'
```

Comment:
theres no `dev` for analytics, we could however create a different tag or siteId to separate